### PR TITLE
Show menu's close arrow next to resizer handler

### PR DESCRIPTION
**Pull request by aleixsuau**
https://community.openproject.com/projects/openproject/work_packages/details/33187

Show the main menu's close arrow button next to resizer handler so the user can resize or hide the menu.

### CODE REVIEW

File: app/assets/stylesheets/layout/_main_menu.sass [hunk 1]
```diff
@@ -345,73 +360,45 @@ a.main-menu--parent-node
...
-    line-height: 40px
     &:before
-      color: $light-gray
-      padding-right: 0
-      padding-left: 2px
+      font-family: openproject-icon-font
+      font-size: 11px
+      font-style: normal
+      text-transform: none
+      font-weight: 400
```
Comment (HDinger) [inline app/assets/stylesheets/layout/_main_menu.sass]: You can replace most of this block with the mixing `icon-common`. It is used every time an icon needs to be added via a .sass-file. I assume that only the `font-size` is still needed.

**Review by HDinger — COMMENTED** [review]
Looks good, I like the change. I only have a minor remark. Otherwise, good to merge 👍

File: app/assets/stylesheets/layout/_main_menu.sass (hunk 1 above)
Comment (aleixsuau) [reply app/assets/stylesheets/layout/_main_menu.sass]: Thank you for the advice @HDinger, I hope it is all right now.

**Review by HDinger — APPROVED** [review]
👍

**Pull request merged**